src/PscIde/Server.purs

```diff
@@ -46,7 +47,8 @@ type PscIdeServerArgs = {
 
 defaultServerArgs :: PscIdeServerArgs
 defaultServerArgs = {
-  exe: "psc-ide-server",
+  exe: "purs",
+  combinedExe: true,
```

Comment:
Sure, I want to support an overlap for a while (will put a setting in the editor) but yeah at some point it will be obsolete